src/integrations/admin/social-templates-integration.php

```diff
@@ -11,6 +11,7 @@
 use Yoast\WP\SEO\Conditionals\Open_Graph_Conditional;
 use Yoast\WP\SEO\Presenters\Admin\Badge_Presenter;
 use Yoast\WP\SEO\Presenters\Admin\Premium_Badge_Presenter;
+use Yoast\WP\SEO\Presenters\Admin\Alert_Presenter;
```

Comment:
I see usually the `use` statements are in alphabetical order, but not use if it's an actual policy...